src/main/java/Lowercase.java

```diff
@@ -0,0 +1,14 @@
+public class Lowercase implements TextFormatter{
+
+    String text;
+
+    public Lowercase (String text) {
+        this.text = text;
+    }
+
+    @Override
+    public String formatText(String text) {
+    return text.toLowerCase();
```

Comment:
jeżeli zrobię tak:
```
new Lowercase("blabla").formatText("test123") 
```

to co zostanie sformatowane? I po co nam to drugie? ;)

Reply:
Ta część zostanie sformatowana: formatText("test123"). To drugie w takim razie jest nie potrzebne :)